istio-control/istio-config/templates/_podDisruptionBudget.tpl

```diff
@@ -0,0 +1,8 @@
+{{- define "podDisruptionBudget.spec" }}
```

Comment:
I'm  trying to minimize templating - you can just inline it, it's not used in any other place.